installing/install_config/installing-customizing.adoc

```diff
@@ -6,19 +6,23 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-Although directly making changes to {product-title} nodes is discouraged,
-there are times when it is necessary to implement a
-required low-level security, redundancy, networking, or performance feature.
-Direct changes to {product-title} nodes can be done by:
+{product-title} supports both cluster-wide and per-machine configuration via Ignition,
+which allows arbitrary partitioning and file content changes to the operating system.
+In general, if a configuration file is documented in {op-system-base-full}, then modifying
+it via Ignition is supported.
+
+There are two ways to deploy machine config changes:
 
 * Creating machine configs that are included in manifest files
 to start up a cluster during `openshift-install`.
 
 * Creating machine configs that are passed to running
 {product-title} nodes via the Machine Config Operator.
 
-* Creating an Ignition config that is passed to `coreos-installer`
-when installing bare-metal nodes.
+Additionally, modifying the "pointer ignition" configuration, for example
```

Comment:
We don't use the term pointer configuration in the docs, is there another term here that makes sense to use? Or maybe a rewording:
```suggestion
Additionally, modifying the reference config, such as
```